src/basho_bench_duration.erl

```diff
@@ -96,20 +96,32 @@ terminate(Reason, #state{duration=DurationMins}) ->
         WSup ->
             WRef = erlang:monitor(process, WSup),
             supervisor:terminate_child(basho_bench_run_sup, basho_bench_worker_sup),
+            supervisor:terminate_child(basho_bench_run_sup, basho_bench_stats),
             receive
                 {'DOWN', WRef, process, _Object, _Info} ->
                     ok
             end
     end,
     run_hook(basho_bench_config:get(post_hook, no_op)),
+    %% If eprof_file is not empty, run eprofiling and write output to filename
+    case EprofFile = basho_bench_config:get(eprof_file, "") of 
```

Comment:
In general you want to use atoms like `undefined` as default values to match on rather than values like `""`.

The usage of assignment inside the case expression is somewhat odd. It doesn't look like you're using the EprofFile outside of the case clause, so I would just move it in like so:

```erlang
case basho_bench_config:get(eprof_file, undefined) of
    undefined ->
        ok;
    EprofFile ->
        ...
end
```